### PR TITLE
Fix for issue 328

### PR DESCRIPTION
**Pull request by r-sreesaran**

### CODE REVIEW

File: openwebstart/src/main/java/com/openwebstart/controlpanel/SimpleDesktopSettingsPanel.java [hunk 1]
```diff
@@ -42,6 +45,7 @@ public SimpleDesktopSettingsPanel(DeploymentConfiguration config) {
             config.setProperty(shortcutComboOptions.getActionCommand(), c.getValue());
         });
 
+        uiLock.update(OwsDefaultsProvider.REMOTE_DEBUG, shortcutComboOptions);
```
Comment (sclassen) [inline openwebstart/src/main/java/com/openwebstart/controlpanel/SimpleDesktopSettingsPanel.java:48]: For me it looks like the `OwsDefaultsProvider.REMOTE_DEBUG` is a copy-paste error.
Can you change this to the correct key please

File: openwebstart/src/main/java/com/openwebstart/controlpanel/SimpleDesktopSettingsPanel.java [hunk 2]
```diff
@@ -1,6 +1,8 @@
 package com.openwebstart.controlpanel;
 
+import com.openwebstart.config.OwsDefaultsProvider;
```
Comment (sclassen) [inline openwebstart/src/main/java/com/openwebstart/controlpanel/SimpleDesktopSettingsPanel.java:3]: This Import should also fall away when the other thing has been fixed

**Review by sclassen — CHANGES_REQUESTED** [review]

**Comment by r-sreesaran** [comment]
review comments adapted

**Comment by sclassen** [comment]
Thank you for the PR and the fast fixing of the comments !!

**Review by sclassen — APPROVED** [review]

**Pull request merged**